tour/basic-read-write.md

```diff
@@ -1,5 +1,67 @@
 ---
-title: Basic Reading & Writing
+title: Writing and Reading
 ---
+Accumulo is a big data key/value store.  Writing data to Accumulo is flexible and fast.  Like any database, Accumulo stores
+data in tables and rows.  Each row in an Accumulo table can hold many key/value pairs.  
 
-Talk about reading and writing.
+1. Start by connecting to Mini Accumulo and create a table called "superheroes".  For now, connect as the root user.
+```java
+Connector conn = mac.getConnector("root", "tourguide");
```

Comment:
I would recommend restructuring this so that the code is all together and can be copied and pasted.  Anything explanation could be comments in the code.